### PR TITLE
Logging utils and config

### PR DESCRIPTION
**Pull request by tstescoTT**
# change log
- move logging_utils.py to utils/logging_utils.py
- add set_vllm_logging_config() to add logging to timestamped logfile as well as to stdout using vLLM logging config

### CODE REVIEW

File: vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py [hunk 1]
```diff
@@ -30,11 +27,22 @@ def get_encoded_api_key(jwt_secret):
...
+    print(f"setting vllm logging config at: {config_path}")
+    print(f"setting vllm logging file at: {log_path}")
+    # note: the vLLM logging environment variables do not cause the configuration
+    # to be loaded in all cases, so it is loaded manually in set_vllm_logging_config
+    os.environ["VLLM_CONFIGURE_LOGGING"] = "1"
+    os.environ["VLLM_LOGGING_CONFIG"] = str(config_path)
+    # stop timeout during long sequential prefill batches
+    # e.g. 32x 2048 token prefills taking longer than default 30s timeout
+    # timeout is 3x VLLM_RPC_TIMEOUT
+    os.environ["VLLM_RPC_TIMEOUT"] = "200000"  # 200000ms = 200s
```
Comment (milank94) [inline vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py:42]: Does it make sense to put this in the `.env` file that gets passed during the launch? Along with the other variables such as `WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml`

Comment (tstescoTT) [reply vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py:42]: linking to conversation at https://github.com/tenstorrent/tt-inference-server/pull/45#discussion_r1862878924

I'd recommend keeping this here for now, and refactor out into a separate config file that we can keep in version control.

**Review by milank94 — APPROVED** [review]

File: vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py [hunk 2]
```diff
@@ -30,11 +27,22 @@ def get_encoded_api_key(jwt_secret):
     return encoded_jwt
 
 
-@patch.object(LLMEngine, "__init__", new=logging_init_wrapper)
```
Comment (mvanniasingheTT) [inline vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py:33]: I may be missing something, but without using the wrapper, how is the raw logging class for the api server being initialized?

Comment (tstescoTT) [reply vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py:33]: We will use it for benchmarking only. We're not enabling the stats logging by default in the main run script.

**Pull request merged**